src/include/ndpi_typedefs.h

```diff
@@ -818,48 +818,6 @@ struct ndpi_lru_cache {
 struct ndpi_flow_tcp_struct {
   /* TCP sequence number */
   u_int32_t next_tcp_seq_nr[2];
-  u_int16_t last_tcp_pkt_payload_len;
-
-  /* Part of the TCP header */
-  u_int8_t cli2srv_tcp_flags, srv2cli_tcp_flags;
-  u_int32_t seen_syn:1, seen_syn_ack:1, seen_ack:1;
```

Comment:
@0xA50C1A1, could you try to keep (all) the TCP fields at the very beginning of the structure?

Reply:
Ok, let's try this way...